cpp/src/ray/config_internal.h

```diff
@@ -24,7 +24,7 @@ class ConfigInternal {
 
   int node_manager_port = 0;
 
-  std::string dynamic_library_path = "";
+  std::vector<std::string> code_search_path;
 
   std::string plasma_store_socket_name = "";
 
```

Comment:
For the `Instance()` method below, can we return `const ConfigInternal &`?

Reply:
I can't make it a const now because the config instance is mutable in current design. Maybe I can make it better in 
a future independent PR.